contrib/vagrant/scripts/03-install-kubernetes-worker.sh

```diff
@@ -31,13 +31,11 @@ EOF
 function install_containerd() {
```

Comment:
What about the `02-install-kubernetes-master` file?

Reply:
That script downloads some files, but they're executables and don't need to be decompressed. I don't have a good way to check their integrity.

Reply:
@pchaigno definitely not blocking, k8s has these urls available
```
# for kubectl
https://dl.k8s.io/v1.17.1/kubernetes-client-linux-amd64.tar.gz
# for kube-apiserver, kube-controller-manager and kube-scheduler
https://dl.k8s.io/v1.17.1/kubernetes-server-linux-amd64.tar.gz
# for kube-proxy and kubelet
https://dl.k8s.io/v1.17.1/kubernetes-node-linux-amd64.tar.gz
```